lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -305,9 +305,11 @@ def self.resolve_automation_object(uri, user_obj, attr = nil, options = {}, read
 
   def self.ae_user_object(options = {})
     raise "user_id not specified in Automation request" if options[:user_id].blank?
-    # raise "group_id not specified in Automation request" if options[:miq_group_id].blank?
+    raise "miq_group_id not specified in Automation request" if options[:miq_group_id].blank?
```

Comment:
@gmcculloug ```miq_group_id``` was not required because a user's ```current_group``` is used as default.
Seems ```AutomationTask``` is not passing ```miq_group_id``` when calling into automate: [here](https://github.com/ManageIQ/manageiq/blob/731174c7c4cb6f21a61c8f3270c5fca5d6513866/app/models/automation_task.rb#L25). Will fix it in another PR.